includes/definitions.inc.php

```diff
@@ -2228,6 +2228,11 @@
 $config['os'][$os]['over'][2]['graph'] = 'device_mempool';
 $config['os'][$os]['over'][2]['text']  = 'Memory Usage';
 
+$os = 'trapeze';
```

Comment:
That's fine as long as trapeze is the os name, the firmware name or if neither of those is what they are known as. We try and avoid using hardware / vendor names as if they release multiple device types (say routers) then we can't have the same name for both types.